pages/organiser.tsx

```diff
@@ -0,0 +1,28 @@
+import { FC } from 'react';
+import { Col, Container, Row } from 'react-bootstrap';
+
+import { PersonCard } from '../components/PersonCard';
+import * as data from './api/data';
+
+const organizers = Object.values(data).flatMap(({ organiser }) => organiser);
+
+const Organiser: FC = () => (
+  <Container>
+    <h1 className="mb-4">社区组织者</h1>
+    <Row as="ul" className="list-unstyled text-center g-5" xs={2} sm={6}>
+      {organizers.map(({ name, link, pic }) => (
+        <Col key={name} className="">
```

Comment:
```suggestion
    <PageHead title="社区组织者">
    <h1 className="mb-4">社区组织者</h1>

    <Row as="ul" className="list-unstyled text-center g-5" xs={2} sm={6}>
      {organizers.map(({ name, link, pic }) => (
        <Col key={name}>
```

每个页面都需要设置好标签页标题栏，参考：
https://github.com/freeCodeCamp-China/city/blob/9dba053209aa5aa132d432d9af65948765e27b81/pages/component.tsx#L55-L58